.github/workflows/test.yml

```diff
@@ -87,10 +97,19 @@ jobs:
         # This job uses a cmd shell, therefore the environment variable syntax is different!
         # The ">-" in the next line replaces newlines with spaces (see https://stackoverflow.com/a/66809682).
         run: >-
-          python -m pytest -vv --cov=pymc_extras --cov-append --cov-report=xml --cov-report term --durations=50 %TEST_SUBSET%
+          python -m pytest --color=yes -vv --cov=pymc_extras --cov-append --cov-report=xml --cov-report term --durations=50 %TEST_SUBSET%
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v2
         with:
           env_vars: TEST_SUBSET
           name: ${{ matrix.os }}
           fail_ci_if_error: false
+
+  all_tests:
+    if: ${{ always() }}
+    runs-on: ubuntu-latest
+    needs: [ ubuntu, windows ]
+    steps:
+      - name: Check build matrix status
```

Comment:
I don't know why we use this name elsewhere doesn't make any sense

Reply:
Wht do you recommend. Make a suggestion

Reply:
Check all tests pass

Reply:
Done